source/diffHandler.py

```diff
@@ -70,31 +85,23 @@ def diff(self, newText: str, oldText: str) -> List[str]:
 				return []
 			with DiffMatchPatch._lock:
 				self._initialize()
-				old = oldText.encode("utf-8")
-				new = newText.encode("utf-8")
 				# Sizes are packed as 32-bit ints in native byte order.
 				# Since nvda and nvda_dmp are running on the same Python
 				# platform/version, this is okay.

```

Comment:
are these comments still accurate?

Reply:
Yes, this comments about packing integers, this code has not changed